searcher/go.sum

```diff
@@ -1,20 +1,28 @@
+github.com/bbalet/stopwords v1.0.0 h1:0TnGycCtY0zZi4ltKoOGRFIlZHv0WqpoIGUsObjztfo=
```

Comment:
There seems to be an issue with the `go mod` in this repo, not a blocker for this particular MR but we should address this in some other MR.